src/components/ProductCard.tsx

```diff
@@ -1,42 +1,18 @@
 import React, { ReactNode } from "react"
-import { IGatsbyImageData } from "gatsby-plugin-image"
-import { gql,useQuery } from "@apollo/client"
 import {
   Badge,
   Box,
   Center,
   Flex,
   Heading,
   HStack,
-  Img,
   TextProps,
 } from "@chakra-ui/react"
 
 import { ButtonLink } from "./Buttons"
-import GatsbyImage from "./GatsbyImage"
-import GitStars from "./GitStars"
+import { Image } from "@/components/Image"
 import Text from "./OldText"
-
-const REPO_DATA = gql`
```

Comment:
Removed this functionality because it is broken in production, so its something we can probably add in later if we want.